### PR TITLE
【Developer Tutorials】fix useEffect()

### PR DESCRIPTION
**Pull request by mashharuki**
add connect() function.

### Description

add connect() function before account() function.

### Test Plan
I tried aptos's tutorial  「Your First Dapp」and `window.aptos.account()` in the app didn't work well.
So I added `connect()` function before `account()` function.

Below is the link of my github repo.

[https://github.com/mashharuki/AptosRepo](https://github.com/mashharuki/AptosRepo)

 Pleae try the following to test.

1. `cd first-dapp`
2. `npm i`
3. `npm run start`
4.  please access to [http://localhost:3000](http://localhost:3000)

You can see a dapp like this.
スクリーンショット 2022-10-12 16 46 07

If you push "送信" button , a popup will appear.
スクリーンショット 2022-10-12 16 47 23

- - -
This change is [Reviewable](https://reviewable.io/reviews/aptos-labs/aptos-core/4988)

### CODE REVIEW

**Comment by manhhuycao** [comment]
Hi mashharuki,
Thank you for shared.

I have a problem connect with aptos --> await window.aptos.connect(); Error: Uncaught (in promise) TypeError: Cannot read properties of undefined (reading 'connect')
 
This's my code:


import React from 'react';
import './App.css';
import { useState,useEffect } from 'react';

function App() {

  const [address, setAddress] = useState<string | null>(null);
  
  /**
   * init function
   */
   const init = async() => {
    // connect
    await window.aptos.connect();
    const data = await window.aptos.account(); 
    // set address
    setAddress(data.address);
  }

  // hook
  useEffect(() => {
    init();
  }, []);
  
  return (
    <div className="App">
      <header className="App-header">
        
        <button
           onClick={init}
        >Connect wallet</button>
        <h3>Wallet address: {address}</h3>


      </header>
    </div>
  );
}

export default App;

So happy if you help me find the bug.
Thank you.

File: developer-docs-site/docs/tutorials/first-dapp.md [hunk 1]
```diff
@@ -114,8 +114,20 @@ import './App.css';
...
+  
+  /**
+   * init function
+   */
+  const init = async() => {
+    // connect
+    await window.aptos.connect();
+    const data = await window.aptos.account(); 
+    // set address
+    setAddress(data.address);
```
Comment (hariria) [inline developer-docs-site/docs/tutorials/first-dapp.md:126]:
```suggestion
    // connect
    const { address, publicKey } = await window.aptos.connect();
    setAddress(address);
```

**Review by hariria — COMMENTED** [review]
Hey thanks for bringing this to our attention, great suggestion. I added a suggestion on how we could improve on it. Lmk what you think

**Review by hariria — APPROVED** [review]
LGTM

**Pull request merged**